src/other/Table.js

```diff
@@ -469,18 +503,22 @@
             }
             this.fixedCol
                 .style("position", "absolute")
-                .style("margin-top", -this.tableDiv.node().scrollTop + tableMarginHeight + "px")
+                .style("margin-top", -this.tableDiv.node().scrollTop + parseInt(tableMarginHeight) + "px")
             ;
             fixedColTd
                 .style("width", fixedColWidth + "px")
             ;
             this.fixedColHead
                 .style("position", "absolute")
-                .style("margin-top", (this.fixedHeader() ? this.tableDiv.node().scrollTop: 0) - tableMarginHeight + "px")
+                .style("margin-top", (this.fixedHeader() ? this.tableDiv.node().scrollTop: 0) - parseInt(tableMarginHeight) + "px")
             ;
             fixedColTh
                 .style("width", fixedColWidth + "px")
             ;
+            rows.each(function(d, i) {
+                var height = d3.select(this).select("td").node().offsetHeight;
```

Comment:
@dtsnell4 This selector will grab unexpected elements when a Table widget exists within another Table widget.